ipykernel/kernelbase.py

```diff
@@ -5,21 +5,20 @@
 from __future__ import annotations
 
 import asyncio
-import concurrent.futures
 import inspect
 import itertools
 import logging
 import os
-import socket
+import queue
```

Comment:
There are multiple imports of `queue`, this line and line 20. Just keep the latter?

Reply:
> Just keep the latter?

Actually Ruff should have removed line 20 since it's unused?
But I agree we should only keep line 20.

Reply:
Removed the dupe